shared/actions/profile/index.tsx

```diff
@@ -170,6 +170,30 @@ const backToProfile = (state: TypedState) => [
   Tracker2Gen.createShowUser({asTracker: false, username: state.config.username}),
 ]
 
+const wotVouch = async (action: ProfileGen.WotVouchPayload, logger: Saga.SagaLogger) => {
+  const {otherText, proofs, statement, username, verificationType} = action.payload
+  try {
+    await RPCTypes.wotWotVouchRpcPromise(
+      {
+        assertion: username,
+        confidence: {
+          other: otherText,
+          proofs,
+          usernameVerifiedVia: verificationType,
+        },
+        vouchTexts: [statement],
+      },
+      Constants.wotAuthorWaitingKey
+    )
+  } catch (e) {
+    logger.warn('Error from wotVouch:', e)
+    return ProfileGen.createWotVouchSetError({
+      error: e.desc || `There was an error submitting the claim.`,
+    })
+  }
+  return [ProfileGen.createWotVouchSetError({error: ''}), RouteTreeGen.createClearModals()]
```

Comment:
Would this line indicate a successful RPC call?

Optional:

Why dispatch an empty error action here? If the intent is to clear out errors in the store and clear modals, consider creating a new action in `actions/json/profile` just for success, like `wotVouchSuccess`.

Reply:
This line indicates a successful RPC call. Is there something fishy about that?

The intent is to clear out errors in the store and close that modal. Why have two action types instead of one?